homeassistant/components/cover/wink.py

```diff
@@ -21,6 +21,10 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
         _id = shade.object_id() + shade.name()
         if _id not in hass.data[DOMAIN]['unique_ids']:
             add_devices([WinkCoverDevice(shade, hass)])
+    for shade in pywink.get_shade_groups():
+        _id = shade.object_id() + shade.name()
+        if _id not in hass.data[DOMAIN]['unique_ids']:
```

Comment:
Sidenote: We should not use the `DOMAIN` name unless it's actually the domain of the platform, ie in this case that's `'cover' `. You can import the wink domain as `WINK_DOMAIN`.